stdlib/2and3/traceback.pyi

```diff
@@ -82,7 +88,7 @@ if sys.version_info >= (3, 5):
 
 
 if sys.version_info >= (3, 5):
-    class StackSummary:
+    class StackSummary(List):
```

Comment:
Oh, it's actually a list of FrameSummary objects, so you should inherit from `List[FrameSummary]`.

It seems like the CPython documentation is wrong here; I filed http://bugs.python.org/issue30984 to fix it.